### PR TITLE
added clarification to the kubeadm reset command

### PR DESCRIPTION
**Pull request by Klaven**
updated the docs to reflect `kubeadm reset`'s best effort nature.

xref#https://github.com/kubernetes/kubeadm/issues/1715

### CODE REVIEW

**Comment by sftim** [comment]
This looks like generated documentation; if that's right, the fix needs to go upstream first.

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-Run this to revert any changes made to this host by 'kubeadm init' or 'kubeadm join'
+Performs a best effort revert of changes made to this host by 'kubeadm init' or 'kubeadm join'
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]: @Klaven the change needs to be in https://github.com/kubernetes/website/blob/master/content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md#reset-workflow-reset-workflow and not the generated file.

Comment (Klaven) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]: :man_facepalming: fixed

**Comment by neolit123** [comment]
thanks
/lgtm
/sig cluster-lifecycle

**Pull request merged**